questionnaires/chair_rises/chair_rises_armt.json

```diff
@@ -0,0 +1,403 @@
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "info-type",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "chair_rises_completed",
+        "form_name": "chair_rises",
+        "section_header": "",
+        "field_type": "yesno",
+        "field_label": "Were you able to complete the most recent test?",
+        "select_choices_or_calculations": "",
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "chair_rises_datetime",
+        "form_name": "chair_rises",
+        "section_header": "",
+        "field_type": "text",
+        "field_label": "Please enter the date and time that you started the chair rises test.",
+        "select_choices_or_calculations": "",
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "datetime_dmy",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "[were_you_able_to_complete] = '1'",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "",
+        "evaluated_logic": "responses['were_you_able_to_complete'] ==  '1'"
+    },
+    {
+        "field_name": "chair_rises_borg_scale",
+        "form_name": "chair_rises",
+        "section_header": "",
+        "field_type": "radio",
+        "field_label": "Please select the option that best describes your level of exertion during the test.",
+        "select_choices_or_calculations": [
+            {
+                "code": "0",
+                "label": "Nothing at all"
+            },
+            {
+                "code": "1",
+                "label": "Very light"
+            },
+            {
+                "code": "2",
+                "label": "Light"
+            },
+            {
+                "code": "3",
+                "label": "Moderate"
+            },
+            {
+                "code": "4",
+                "label": "Somewhat Hard"
+            },
+            {
+                "code": "5",
+                "label": "Hard"
```

Comment:
I think the labels should also contain the numbers like `5 Hard` so it is consistent with the scale as some labels contain just numbers